package.json

```diff
@@ -212,7 +212,7 @@
 		"winston": "^3.8.2"
 	},
 	"devDependencies": {
-		"@aws-sdk/client-s3": "^3.44.0",
+		"@aws-sdk/client-s3": "^3.352.0",
```

Comment:
this is not an upgrade, but a downgrade...

Reply:
No, this is upgrade last version is 3.360.0
And 3.44.0 was for 2 years ago